src/usb_cam.cpp

```diff
@@ -314,17 +314,19 @@ void uyvy2rgb(char *YUV, char *RGB, int NumPixels)
   }
 }
 
-static void mono102mono8(char *RAW, char *MONO, int NumPixels)
+static void mono102mono8(unsigned char *RAW, char *MONO, int NumPixels)
```

Comment:
What's the rationale for only making `RAW` unsigned and not `MONO`? This also applies to the other conversion functions.